src/test/java/com/philvarner/clamavj/test/ClamScanTestCase.java

```diff
@@ -79,12 +117,13 @@ public void testTooLarge() throws Exception {
         assertEquals(ScanResult.RESPONSE_SIZE_EXCEEDED, result.getResult());
     }
 
+    @Test
     public void testStats() throws Exception {
         String result = scanner.stats();
-        assertTrue("didn't contain POOLS: \n" + result, result.contains("POOLS:"));
-        assertTrue("didn't contains STATE: \n" + result, result.contains("STATE:"));
+        assertTrue("didn't contain POOLS: \n" + result, result.contains("POOLS:") || result.contains("STATE:"));
```

Comment:
should the logical operator here be && instead of || to match what was in the original test?

Reply:
This wasn't a change I made, it was pulled in from the original repo, so hopefully they had a reason for it